cmd/ferretdb/main.go

```diff
@@ -85,24 +90,43 @@ func main() {
 
 	go debug.RunHandler(ctx, *debugAddrF, logger.Named("debug"))
 
-	pgPool, err := pgdb.NewPool(ctx, *postgresqlURLF, logger, false)
-	if err != nil {
-		logger.Fatal(err.Error())
+	var h common.Handler
+
+	startTime := time.Now()
```

Comment:
There is only one usage of that variable. We can call time.Now() right in `handlerOpts` initialization as we do in `integration/setup.go`.

Reply:
moved